projects/components/src/utils/test/widget-object/cypress/cypress-widget-finder.ts

```diff
@@ -40,11 +40,23 @@ export class CypressWidgetObjectFinder<T> {
         if (findOptions?.cssSelector) {
             tagName += `${findOptions.cssSelector}`;
```

Comment:
Don't you need to prefix with a space if you're going to concat the two selectors?

For example:
```tagName += ` ${findOptions.cssSelector}`;```

Reply:
Same as in the below discussion.